models/spring-ai-openai/src/main/java/org/springframework/ai/openai/OpenAiChatModel.java

```diff
@@ -143,8 +143,7 @@ public class OpenAiChatModel extends AbstractToolCallSupport implements ChatMode
 	 * @throws IllegalArgumentException if openAiApi is null
 	 */
 	public OpenAiChatModel(OpenAiApi openAiApi) {
-		this(openAiApi,
-				OpenAiChatOptions.builder().withModel(OpenAiApi.DEFAULT_CHAT_MODEL).withTemperature(0.7).build());
+		this(openAiApi, OpenAiChatOptions.builder().model(OpenAiApi.DEFAULT_CHAT_MODEL).temperature(0.7).build());
```

Comment:
we should check if these defaults correspond to what the autoconfig defaults are